subprojects/testing-jvm/src/test/groovy/org/gradle/api/tasks/testing/TestTest.groovy

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.tasks.testing
+
+import org.gradle.api.tasks.Nested
+import org.gradle.api.tasks.Optional
+import org.junit.Test
+
+import static org.junit.Assert.*
+
+class TestTest {
```

Comment:
Since when do we write regular junit tests instead of spock tests? I like JUnit but it seems pretty off given almost no other test uses JUnit (except 9 outliers)

Reply:
And using JUnit without AssertJ leads to not so pretty tests in my eyes

Reply:
Because of the siblings of `GroovyCompileTest` but you are right, will update to Spock tests.